playground/src/model/module_response.ts

```diff
@@ -2,6 +2,7 @@ type ModuleResponse = {
   module_name: string;
   status: number;
   data: any;
+  meta: any;
```

Comment:
```suggestion
  meta?: any;
```
Does this make all the `meta` changes unnecessary?